src/python/pants/backend/project_info/tasks/templates/idea/project-12.mustache

```diff
@@ -93,18 +93,6 @@ Licensed under the Apache License, Version 2.0 (see LICENSE).
   {{/project.git_root}}
   </component>
 
-  <component name="CheckStyle-IDEA">
-    <option name="configuration">
-      <map>
-        <entry key="active-configuration" value="file://{{project.root_dir}}/build-support/checkstyle/coding_style.xml:Twitter CheckStyle rules." />
-        <entry key="check-test-classes" value="test" />
-        <entry key="location-0" value="CLASSPATH:/sun_checks.xml:The default CheckStyle rules." />
-        <entry key="location-1" value="file://{{project.root_dir}}/build-support/checkstyle/coding_style.xml:Twitter CheckStyle rules." />
-        <entry key="thirdparty-classpath" value="{{project.checkstyle_classpath}}" />
-      </map>
-    </option>
-  </component>
-
```

Comment:
Is this change something that had to be done to upgrade the IDEA plugin version?

Reply:
No, it is not. Reverted. Thanks!